src/components/toast/ToastReducers.ts

```diff
@@ -10,12 +10,8 @@ export interface IToastsState {
 
 export interface IToastState {
     id: string;
-    title: string;
-    type?: string;
-    dismiss?: number;
-    dismissible?: boolean;
-    animate?: boolean;
-    content?: React.ReactNode;
+    title: React.ReactNode;
+    [toastProp: string]: any;

```

Comment:
Can you do something like `IToastState extends IToastProps` instead ? Otherwise I think we will lose a lot of type completion with this

If it doesn't work maybe something like `[toastProp: KeyOf<IToastProps>]: any;`